ui/webui/test/anacondalib.py

```diff
@@ -27,11 +27,25 @@
 from machine_install import VirtInstallMachine
 from testlib import MachineCase  # pylint: disable=import-error
 
+from storage import Storage
+
 
 class VirtInstallMachineCase(MachineCase):
     MachineCase.machine_class = VirtInstallMachine
 
     def setUp(self):
+        # FIXME: running this in destructive tests fails because the SSH session closes before this is run
+        if self.is_nondestructive():
+            self.addCleanup(self.rescanStorage)
```

Comment:
Maybe we could print/log a warning when someone calls this by mistake in destructive mode ?

Reply:
Hmm i don't see how I can know if rescanStorage will be added with an addCleanup in the future